src/Bloc/BlMatrixTransformation.class.st

```diff
@@ -19,6 +19,10 @@ BlMatrixTransformation class >> matrix: aBlMatrix [
 		  yourself
 ]
 
+{ #category : #comparing }
+BlMatrixTransformation >> = anObject [
+]
```

Comment:
This returns self, not a Boolean! it should compare the `matrix` instvar, right?

Reply:
I'm sorry I didn't see it before merging it @Nyan11

Reply:
Oups, it should not be there. I didn't do equality for Transformation. Only for transformation origin

Reply:
I fix it in https://github.com/pharo-graphics/Bloc/pull/473